### DISCUSSION TITLE
How long takes generally for helix to update the packages?

### DISCUSSION THREAD

**Discussion by norman784** [helix-editor/helix#1480] [Q&A] [answered]
In my case I'm curious about Homebrew, but it seems that it takes a while for helix to be updated, is there a better way to have  the editor up to date?

Also does the helix team update some package distribution system directly? (example is Macports/Homebrew updated directly by the team?)

**Comment by archseer** [comment, marked as answer]
We have our own tap that I forgot to update until now, it should be up to date now: https://github.com/helix-editor/homebrew-helix/commit/719f09c53e107fb5d6f6a0670ffb66bc581fff32

Generally it depends on the distribution since we don't have control over downstream packaging.

**Comment by norman784** [comment]
Cool, thanks!

**Comment by dylrich** [comment]
This is an old discussion and perhaps not possible for you, but I have never once experienced a broken master and I've built Helix from source every day or two for the past 7ish months on multiple machines. I'd highly recommend just keeping a local clone, pulling periodically, and re-running `cargo install --path=helix-term`. You also have to check for updated submodules fairly often due to languages being added/changed around, but it's not too bad. I usually run the following:

```bash
git pull && git submodule sync; git submodule update --init
cargo install --path=helix-term
rm -rf ~/.config/helix/runtime && cp -r runtime ~/.config/helix
```
